doc/nrf/libraries/others/dult.rst

```diff
@@ -44,6 +44,25 @@ The following Kconfig options are also available for this module:
   * :kconfig:option:`CONFIG_DULT_BT_ANOS_INDICATION_COUNT` - This option allows to configure the number of simultaneously processed GATT indications by the ANOS.
     The default value is set to ``2``.
 
+* :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` - This option enables support for motion detector.
+  The motion detector is an optional feature in the DULT specification.
+  For more details see the `DULT motion detector`_ section of the DULT specification.
+  This option is disabled by default.
+
+  * :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_TEST_MODE` - This option allows to configure motion detector parameters for testing purposes.
+    These values are defined in the DULT specification and should not be changed in the production code.
+    This option is disabled by default.
+
+    * :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_SEPARATED_UT_BACKOFF_PERIOD` - This option allows to configure the period in minutes to disable the motion detector if the accessory is in the separated state.
```

Comment:
Take these three bullets back two steps. I guess they can be on the same level as the one above.

Reply:
They have prompt only if the `CONFIG_DULT_MOTION_DETECTOR_TEST_MODE` is enabled so I wanted to group them under this kconfig description.